apps/api/src/server/applications/examination-timetable-items/__tests__/examination-timetable-items.test.js

```diff
@@ -1,11 +1,22 @@
 import { jest } from '@jest/globals';
 import { request } from '../../../app-test.js';
```

Comment:
this is a comment only for future, not a required change now:
I have started to use the # shortcuts where appropriate, and changing them if I change a file - eg import { request  } from '../../../app-test.js'; can be changed to import { request  } from '#app-test.js';
likewise '../../../utils/database-connector.js' can be changed to '#utils/database-connector.js', and same for #infrastructure etc

Reply:
I'll check that from now on, but I'll leave it today